workspace/spec/lita/handlers/teamcity_spec_anki.rb

```diff
@@ -195,6 +195,76 @@
       expect(output_string).to eq(expected_output)
     end
   end
+  
+  describe '#format_time' do
+
+    it 'format_time with time > 3600' do
+      output_string = subject.format_time(3700)
+      expect(output_string).to eq("%Hh:%Mm:%Ss")
+    end
+
+    it 'format_time with time > 60 and <= 3600' do
+      output_string = subject.format_time(3600)
+      expect(output_string).to eq("%Mm:%Ss")
+    end
+
+    it 'format_time with time <= 60' do
+      output_string = subject.format_time(60)
+      expect(output_string).to eq("%Ss")
+    end
+  end
+
+  describe '#queue' do
+    queue_build_json_str = '{ "count": 2,
+                              "build": [{"buildTypeId": "CozmoOne_MasterBuild",
+                                        "branchName": "BI-123-XYZ",
+                                        "id": "4307"},
+                                       {"buildTypeId": "OverDriveOne_Release_Build",
+                                        "branchName": "BI-124-XYZ",
+                                        "id": "4308"}]}'
+    before do
+     stub_request(:get, /buildQueue/).to_return(:status => 200, :body => queue_build_json_str)
+    end
+    
+    it 'Reply all queue builds are found if call "Queue"' do
+      expected_output = "Here are the builds I found:\n"\
+                        "\n<#{registry.config.handlers.teamcity.site}/viewQueued.html?itemId=4307"\
+                        "|CozmoOne_MasterBuild> - BI-123-XYZ"\
+                        "\n<#{registry.config.handlers.teamcity.site}/viewQueued.html?itemId=4308"\
+                        "|OverDriveOne_Release_Build> - BI-124-XYZ"
+      send_command('queue')
+      expect(replies.last).to eq(expected_output)
+    end
+
+    it 'Reply all queue builds are found if call "Queue_Wildcard"' do
+      expected_output = "Here are the builds I found:\n"\
+                        "\n<#{registry.config.handlers.teamcity.site}/viewQueued.html?itemId=4307"\
+                        "|CozmoOne_MasterBuild> - BI-123-XYZ"
+
+      send_command('queue CozmoOne*')
+      expect(replies.last).to eq(expected_output)
+    end
+
+    it 'Reply empty if call "Queue_Wildcard" with invalid wildcard' do
+      expected_output = nil
+      send_command('queue abc')
+      expect(replies.last).to eq(expected_output)
+    end
+    
+    it 'The message display when no have queue is found if call "Queue"' do
```

Comment:
Both the "it 'The message display when no have queue is found if call "Queue"' do" and "it 'Reply empty if call "Queue_Wildcard" with invalid wildcard' do" cases are returned an empty result. So only one case should be tested here